doc/en/developer/source/cite-test-guide/index.rst

```diff
@@ -8,226 +8,72 @@ A step by step guide to the GeoServer Compliance Interoperability Test Engine (C
 .. contents::
    :depth: 2
 
+Requirements
+------------
+
+Running the tests requires a linux system with docker and docker-compose installed on it.
+
 Check out CITE tools
 --------------------
 
-The CITE tools are available at https://github.com/jdeolive/geoserver-cite-tools. The README file 
-contains the most update documentation of how to checkout and build the tools. The quick version is::
+The CITE tools are available in the build/cite folder of the GeoServer repository:
 
-  git clone git://github.com/jdeolive/geoserver-cite-tools.git
-  cd geoserver-cite-tools
-  git submodule update --init
-  mvn clean install
+https://github.com/geoserver/geoserver/tree/master/build/cite
```

Comment:
I should make an `extlink` shortcut for this if we are going to be in the habit of calling out source code locations in our docs.

Reference:

* https://github.com/geoserver/geoserver/blob/master/doc/en/developer/source/conf.py